src/App.test.jsx

```diff
@@ -53,6 +68,11 @@ test('Should render the header with Sasuke 🌬️🔥', async () => {
   }
 
   // 🚨 Use the server to change the response for this test
+  server.use(
+    rest.get('https://uzgiamkrbapxufnwdrja.supabase.co/rest/v1/users', (req, res, ctx) => {
+      return res(ctx.json([sasuke]))
```

Comment:
I would also check for the query params validity here too. When we use msw, we should only feed the base endpoint into rest.get(). But our users.js fetch call actually has a select param baked in. 
If we put the query param into our msw .get() url, it doubles up and throws an error (or warning?). 
But we still want to test for the validity of that query param and make sure to use it. So we do that with an if statement. Check out this doc link: https://mswjs.io/docs/recipes/request-assertions#include-failure-scenarios